packages/deployments/contracts/contracts/Connext.sol

```diff
@@ -33,7 +31,7 @@ import "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeab
 // 1. Finalize BridgeMessage / BridgeRouter structure + backwards compatbility
 // 2. Gas optimizations
 
-contract TransactionManager is Initializable, ReentrancyGuardUpgradeable, ProposedOwnableUpgradeable, MerkleTreeManager {
+contract Connext is Initializable, ReentrancyGuardUpgradeable, ProposedOwnableUpgradeable, MerkleTreeManager {
 
   // TODO: why is this breaking the build
   // uint256 internal constant 3 = 3;
```

Comment:
why *is* this breaking the build?? lol

maybe bc u (or whoever wrote) named the constant `3` ?